script.js

```diff
@@ -27,6 +27,51 @@ crossmenu.addEventListener('click', () => {
   multipost.classList.add('Menu-mobile-after');
 });
 
+// Form Validation L
+const form = document.getElementById('form');
+const emailForm = 'ERROR enter a valid email address';
+
+const hasValue = (input) => {
+  if (input === '') {
+    return false;
+  }
+  return true;
+};
+
+const showMessage = (input, message) => {
+  const msg = input.parentNode.querySelector('small');
+  msg.innerText = message;
+};
+
+const validateEmail = (input, emailForm) => {
+  const value = input.value.trim();
+  if (!hasValue(input.value)) {
+    return false;
+  }
+  const lowerCaseValidator = /[A-Z]/g;
+  if (value.match(lowerCaseValidator)) {
+    input.value = value.toLowerCase();
+    return showMessage(input, emailForm);
+  }
+  return true;
+};
+
+const errorMessage = () => {
+  const allmessages = document.querySelectorAll('.validation');
+  if (allmessages.length > 0) {
+    for (let i = 0; i < allmessages.length; i += 1) { allmessages[i].innerHTML = ''; }
+  }
+};
+
+form.addEventListener('submit', (event) => {
+  const emailSubmit = form.elements.user_email;
+  const validEmail = validateEmail(emailSubmit, emailForm);
+  if (validEmail) {
+    errorMessage();
+  }
+  event.preventDefault();
+});
```

Comment:
- [ ] Your form is expected to submit normally if the email field contains only lowercase characters but your form did not submit. Kindly fix that

![my-gif](https://user-images.githubusercontent.com/16058470/127753742-b185b1f8-8e62-4650-a593-98f0145f949b.gif)